__tests__/models/network.test.js

```diff
@@ -13,28 +13,32 @@ test('Get and set network', () => {
     'mainnet': {
       'p2pkh': 0x28,
       'p2sh': 0x64,
+      'xpriv': 0x03523b05,
```

Comment:
I think we should use:

- Mainnet, xpriv: `Hatr` in ASCII (`0x48617468`)
- Mainnet, xpub: (same as xpriv if possible)
- Testnet, xpriv: `htn!` in ASCII (`0x68746e21`)

What do you think? We should document it properly. Maybe an RFC?

Reply:
I think having the same identifier for xpriv and xpub would generate some confusion, but we could use these for now and discuss on another issue.